sdk/python/feast/infra/gcp.py

```diff
@@ -38,7 +38,9 @@ def compute_datastore_entity_id(entity_key: EntityKeyProto) -> str:
     return mmh3.hash_bytes(serialize_entity_key(entity_key)).hex()
 
 
-def _make_tzaware(t: datetime):
+def _make_tzaware(t: Optional[datetime]):
+    if t is None:
+        return t
```

Comment:
hmm that seems not great from readability perspective, can be explicitly handle `None` in the caller?

In fact, after looking closer, it looks like there is already no chance of this being called with None